thrift/src/main/thrift/atoms/guide.thrift

```diff
@@ -0,0 +1,19 @@
+namespace * contentatom.guide
+namespace java com.gu.contentatom.thrift.atom.guide
+#@namespace scala com.gu.contentatom.thrift.atom.guide
+
+include "../shared.thrift"
+include "entity.thrift"
+
+struct GuideAtom {
+  1: optional string typeLabel
+  2: optional list<string> tags
```

Comment:
tbh I don't know. I just thought we started using tags for explainers in tests and it came to me we might want to do the same here. There is a `labels` field in the Atom struct, but it was always empty in my tests... this may be a problem in the editing UI though?